fvm/evm/types/chainIDs.go

```diff
@@ -10,6 +10,10 @@ var (
 	FlowEVMPreviewNetChainID = big.NewInt(646)
 	FlowEVMTestNetChainID    = big.NewInt(545)
 	FlowEVMMainNetChainID    = big.NewInt(747)
+
+	FlowEVMPreviewNetChainIDInUInt64 = FlowEVMPreviewNetChainID.Uint64()
+	FlowEVMTestNetChainIDInUInt64    = FlowEVMTestNetChainID.Uint64()
+	FlowEVMMainNetChainIDInUInt64    = FlowEVMMainNetChainID.Uint64()
```

Comment:
conversion to uuid is not cheap, so I made it global vars to also save computation per evm tx